config/routes.rb

```diff
@@ -1,15 +1,13 @@
 Rails.application.routes.draw do
   root "users#index"
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-end
-
-# Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-# Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-# Can be used by load balancers and uptime monitors to verify that the app is live.
-get "up" => "rails/health#show", as: :rails_health_check
+    resources :posts, only: [:index, :show, :create] do
```

Comment:
- [ ] [REQUIRED] You have limited the post resources to only `:show`, `:index`, and `:create`. Kindly add the `:new` action

- Kindly check the review summary for the remaining recommended changes